packages/storybook/stories/components/divider/migration.from.17.x.mdx

```diff
@@ -0,0 +1,85 @@
+import { Meta } from '@storybook/blocks';
+import * as DividerStories from './divider.stories';
+
+<Meta of={ DividerStories } name='Migration From 17.x' />
+
+# Divider - migrate from v17 to v18
+----
+
+## Attributes changes
+
+`color`  <img src="https://img.shields.io/badge/updated-00FFFF" />
+
+Has been updated.
+
+The 2 possible values are now `light` and `dark`.
+
+`contrasted ` <img src="https://img.shields.io/badge/removed-FF0000" />
+
+Has been removed.
+
+Color attribute can be use with `dark` for contrasted content.
+
+`seperator ` <img src="https://img.shields.io/badge/removed-FF0000" />
+
+Has been removed.
+
+Divider is always shown by default.
+
+`size ` <img src="https://img.shields.io/badge/removed-FF0000" />
+
```

Comment:
Documentation isn't clear regarding the new attribute, please use the format of the other migration guides to announce the new attribute & specify that Spacing works with `px` values.

Reply:
Description of spacing has been changed